### DISCUSSION TITLE
Lightdm error on xfce and cinnamon

### DISCUSSION THREAD

**Discussion by carlosEgarc** [SpiralLinux/SpiralLinux-project#135] [General]
Hello thanks for you great distro. Today I download  the xfce and later the cinnamon edition. Both stop with the same error "Failed to Start Light Display Manager" im trying to use the live usb in a very old laptop (HP mini Atom N450 2GB ddr3 Ssd 128 Gb) that is running Mint Debian edition (I also test fedora xfce edition ) without problem. But Im want to try  Spiral Linux. Any advice? 

I find this when searching with Google
https://www.debugpoint.com/failed-to-start-lightdm

Thanks 

Carlos

**Comment by geckolinux** [comment]
Hi there, sorry to hear about the trouble.
How did you write SpiralLinux to the USB drive?
Could you please try to boot it with the 5.10 kernel that is also included with SpiralLinux?

**Reply by carlosEgarc** [reply]
Thanks for your time. 

1. Using Mint tool to create usb drive. I test with last Debian 11.5 Xfce live usb image non free (debian-live-11.5.0-amd64-xfce+nonfree.iso)  and is working good.
2. Can you please tell me how to do it ? 

Gracias 

Carlos

**Reply by geckolinux** [reply]
Hi Carlos, does the laptop run in UEFI mode or in legacy BIOS mode?

**Reply by carlosEgarc** [reply]
Thanks. Using BIOS mode i dont find a EFI folder under /sys/firmware/ 

Gracias

**Reply by geckolinux** [reply]
Muy bien, entonces por favor intentemos lo siguiente:
En la primera pantalla de arranque de la sesión live de SpiralLinux presiona la tecla TAB y modifica los parámetros de arranque así y luego presiona ENTER para arrancar:

![Screenshot from 2022-10-02 19-44-13](https://user-images.githubusercontent.com/2692138/193484244-fad70838-b993-4968-9039-65cbda08ee42.jpg)

**Reply by carlosEgarc** [reply]
Gracias por la respuesta en Español. Lamentablemente aún con el kernel 5.10 me muestra el mismo error. Failed to start light Display Mánager.

Gracias 

Carlos

**Reply by geckolinux** [reply]
Hola Carlos, gracias por hacer el intento. Podrías probar ese mismo USB en otra computadora para comprobar si arroja el mismo error?

**Reply by carlosEgarc** [reply]
Si esta trabajando muy bien. HP desktop.

**Reply by geckolinux** [reply]
Gracias, y podrías probar la opción de arranque **Live (amd64 failsafe)** ?
Si no funciona por favor intenta de nuevo pero modificando los párametros con TAB, en especial tratando de arrancarlo con todos los parámetros failsafe _menos_ `vga=788` .

**Reply by carlosEgarc** [reply]
Live (amd64 failsafe)  Quitando la opción vga=788 esta funcionando. Demoro un largo tiempo en cargar pero al final funcionó. Quedó súper lento pero todo se ve bien.

**Reply by geckolinux** [reply]
Gracias Carlos, ya luce un poco mejor la cosa.

 > Demoro un largo tiempo en cargar

Esto es por el párametro failsafe `memtest` que también se puede eliminar.

Tal vez podrías ir experimentando con las opciones que le siguen a `boot=live components` para ver hasta dónde logra arrancarse el sistema con la mínima cantidad de opciones presentes (siempre conservando `initrd=/live/initrd.img` al final). Lamento que sea un proceso un poco aleatorio, pero una vez que logres instalar el sistema al disco duro creo que ya andará sin mayores problemas.

**Reply by carlosEgarc** [reply]
Hola. Siento la demora estaba de vacaciones en Chile. 
En el arranque normal agregando nomodeset elimino el error. Pero la pantalla queda en VGA ( la resolución no es buena). Espero que esta información sirva para corregir el problema. 

Gracias.

Adicionalmente ya instalé pero me toca adicionar en el arranque la misma opción nomodeset para que funcione de lo contrario no arranca.

Con el grub customizer adicione la entrada para que quedara permanentemente. 

Ya actualice el sistema y al momento todo esta trabajando bien. Seguiré las pruebas

Nuevamente gracias.

**Reply by geckolinux** [reply]
Gracias Carlos por confirmarlo. Lamentablemente no creo que se pueda hacer nada por el lado de SpiralLinux. Pero tal vez con la opción `nomodeset` sería suficiente solo para arrancar el sistema live e instalarlo al disco, y una vez instalado pienso que probablemente no tendrá ese problema.
